docs/examples/language_model/language_model.md

```diff
@@ -61,15 +61,17 @@ from mxnet.gluon.utils import download
 import gluonnlp as nlp
 ```
 
-#### Set environment
+#### Setup the environment (note that we should change num_gpus according to how many NVIDIA GPUs are available on the target machine)
 
```

Comment:
The title is too long. Why not move contents within parentheses to the main body before the code?